QgisModelBaker/gui/workflow_wizard/workflow_wizard.py

```diff
@@ -564,13 +564,22 @@ def add_source(self, source, origin_info):
             path = None
         return self.source_model.add_source(name, type, path, origin_info)
 
-    def append_dropped_files(self, dropped_files):
+    def append_dropped_files(self, dropped_files, dropped_ini_files):
         if dropped_files:
             for dropped_file in dropped_files:
                 self.add_source(
                     dropped_file, self.tr("Added by user with drag'n'drop.")
                 )
 
+        if dropped_ini_files:
+            if len(dropped_ini_files) > 1:
+                # TODO logging.warning("Only one INI/TOML file is supported by drag&drop: {}".format(dropped_ini_files))
```

Comment:
Can you do it or just ignore it. But I think it's an easy one...

Reply:
As soon as #814 is merged it can be uncommented